src/util/symbol_table_base.h

```diff
@@ -90,7 +90,7 @@ class symbol_table_baset
   symbolt &get_writeable_ref(const irep_idt &name)
   {
     symbolt *symbol = get_writeable(name);
-    if(symbol == nullptr)
+    if(symbol == nullptr) // consider INVARIANT?
```

Comment:
Maybe? Arguably another case where we want to keep consistency with the standard library, although in this case there's less of a motivation because a symbol table isn't a general purpose container.